<!DOCTYPE html>
<html><head><title>Failed</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">5 references to Failed</div><div class="rA">Azure.Core.Tests (5)</div><div class="rG" id="Azure.Core.Tests"><div class="rF"><div class="rN">ClientDiagnosticsTests.cs (2)</div>
<a href="../../Azure.Core.Tests/ClientDiagnosticsTests.cs.html#172"><b>172</b>scope.<i>Failed</i>(exception);</a>
<a href="../../Azure.Core.Tests/ClientDiagnosticsTests.cs.html#198"><b>198</b>scope.<i>Failed</i>(new Exception());</a>
</div>
<div class="rF"><div class="rN">Shared\OperationInternal.cs (1)</div>
<a href="../../Azure.Core.Tests/Shared/OperationInternal.cs.html#277"><b>277</b>scope.<i>Failed</i>(e);</a>
</div>
<div class="rF"><div class="rN">Shared\SyncAsyncEventHandlerExtensions.cs (1)</div>
<a href="../../Azure.Core.Tests/Shared/SyncAsyncEventHandlerExtensions.cs.html#118"><b>118</b>scope.<i>Failed</i>(ex);</a>
</div>
<div class="rF"><div class="rN">SyncAsyncEventHandlerTests.cs (1)</div>
<a href="../../Azure.Core.Tests/SyncAsyncEventHandlerTests.cs.html#94"><b>94</b>scope.<i>Failed</i>(ex);</a>
</div>
</div>
</body></html>